tox.ini

```diff
@@ -29,3 +29,10 @@ pass_env =
 commands =
     pre-commit run --all-files --show-diff-on-failure
     check-manifest
+
+[testenv:mypy]
+skip_install = true
+deps =
+    mypy==1.7.1
+commands =
+    mypy src {posargs}
```

Comment:
[`{posargs}`](https://tox.wiki/en/4.11.4/user_guide.html#basic-example) isn't used by the Lint job, right? You're imagining this as being helpful locally somehow?

Reply:
That's right, so you can pass arguments through, like `tox -e mypy -- --verbose`.

tox can be useful locally, it takes care of the virtualenv so you're not testing against other things you may have installed globally.